tests/visualization_tests/test_contour.py

```diff
@@ -516,7 +516,6 @@ def test_get_contour_info_nonfinite_multiobjective(objective: int, value: float)
         reverse_scale=True,
         target_name="Objective Value",
     )
-    assert info.sorted_params == ["param_b", "param_d"]
```

Comment:
This is checked in `assert info == _ContourInfo(...` above too, so we can remove this.